Makefile

```diff
@@ -10,24 +13,24 @@ docker-build:
 	docker build --build-arg NGINX_PLUS_VERSION=$(NGINX_PLUS_VERSION)~stretch -t $(NGINX_IMAGE) docker
 
 run-nginx-plus:
-	docker network create --driver bridge test
-	docker run --network=test -d --name nginx-plus --network-alias=nginx-plus-test --rm -p 8080:8080 -p 8081:8081 $(NGINX_IMAGE)
-	docker run --network=test -d --name nginx-plus-helper --network-alias=nginx-plus-test --rm -p 8090:8080 -p 8091:8081 $(NGINX_IMAGE)
+	docker network create --driver bridge $(DOCKER_NETWORK)
+	docker run --network=$(DOCKER_NETWORK) -d --name $(DOCKER_NGINX_PLUS) --network-alias=$(DOCKER_NGINX_PLUS)-test --rm -p 8080:8080 -p 8081:8081 $(NGINX_IMAGE)
```

Comment:
Please note the tests expect that the alias is always `nginx-plus-test` . See https://github.com/nginxinc/nginx-plus-go-client/blob/master/docker/nginx.conf#L59

Reply:
This is fine as local tests should run using default values - the support for parameterisation is primarily to provide the ability to run parallel tests during our build process, where we replace the default alias in the _nginx.conf_.